data/mods/Aftershock/items/magazine/energy_cell.json

```diff
@@ -44,5 +44,21 @@
     "reload_time": 5,
     "flags": [ "NO_UNLOAD" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 600 } } ]
+  },
+  {
+    "id": "afs_archeotech_cartridge",
+    "copy-from": "afs_cartridge",
+    "type": "MAGAZINE",
+    "name": { "str": "chromoelectric cartridge" },
+    "description": "A thin strip of transuranic metal, glowing a somber blue from within its shielded glass prison.  A rare and well sought relic, it could be used to momentarily power all sorts of technology from the late Hyperspace Era.",
```

Comment:
```suggestion
    "description": "A thin strip of transuranic metal, glowing a somber blue from within its shielded glass prison.  A rare and sought-after relic, it could be used to momentarily power all sorts of technology from the late Hyperspace Era.",
```